lexicons/com/atproto/label/subscribeAllLabels.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.label.subscribeAllLabels",
+  "defs": {
+    "main": {
+      "type": "subscription",
+      "description": "Subscribe to label updates",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "cursor": {
+            "type": "integer",
+            "description": "The last known event to backfill from."
+          }
+        }
+      },
+      "message": {
+        "schema": {
+          "type": "object",
+          "required": ["seq", "event", "repo", "commit", "prev", "blocks", "ops", "blobs", "time"],
+          "nullable": ["prev"],
+          "properties": {
+            "seq": {"type": "integer"},
```

Comment:
if a service offers both repo & label subscriptions, the idea is they'd be separate `seq`s right?

Reply:
That is my understanding. the scope of sequences (distinct set of `seq` numbers) would be (host, streamNsid).

I think we want to keep the door open to merging subscription feeds into a unified feed, and I don't think this blocks that, assuming that the merged feed would have a different endpoint. Also seq numbers can in theory have gaps between them, so a unified seq number *could* be used, and then endpoints just filter to the specific type at that endpoint. Haven't thought this through much, just thinking outloud.